thrust/system/detail/generic/sequence.inl

```diff
@@ -61,9 +73,12 @@ __host__ __device__
                 T init,
                 T step)
 {
-  using thrust::placeholders::_1;
 
-  thrust::tabulate(exec, first, last, init + step * _1);
+  thrust::tabulate(exec,
+                   first,
+                   last,
+                   detail::compute_sequence_value<T>{std::move(init),
+                                                     std::move(step)});
```

Comment:
Why was this needed?

Reply:
`_1` is always a `std::size_t` in this context, and there's no way to cast it to the proper type using the placeholder system. This was causing numeric conversion warnings for some values of `T`.